tombstone_gc.cc

```diff
@@ -118,25 +100,67 @@ tombstone_gc_state::get_gc_before_for_range_result tombstone_gc_state::get_gc_be
                 }
             }
             if (hits == 0) {
-                min_repair_timestamp = gc_clock::time_point::min();
-                max_repair_timestamp = gc_clock::time_point::min();
+                min_reconcile_timestamp = gc_clock::time_point::min();
+                max_reconcile_timestamp = gc_clock::time_point::min();
             } else {
                 knows_entire_range = hits == 1 && contains_all;
-                min_repair_timestamp = min;
-                max_repair_timestamp = max;
+                min_reconcile_timestamp = min;
+                max_reconcile_timestamp = max;
             }
-            min_gc_before = check_min(s, saturating_subtract(min_repair_timestamp, propagation_delay));
-            max_gc_before = check_min(s, saturating_subtract(max_repair_timestamp, propagation_delay));
+            min_gc_before = check_min(s, saturating_subtract(min_reconcile_timestamp, propagation_delay));
+            max_gc_before = check_min(s, saturating_subtract(max_reconcile_timestamp, propagation_delay));
```

Comment:
these renames belong to the `tombstone_gc: refactor and simplify the repair map` commit

Reply:
done ... in fact keeping the previous repair code (and added a separate simplified version for the group0), so also kept the "repair" part there for the actual repair code